controllers/managedfusionoffering_controller.go

```diff
@@ -247,6 +275,10 @@ func (r *ManagedFusionOfferingReconciler) getCSVByPrefix(name string) (*opv1a1.C
 
 // All the below functions are placeholder for offering plugin integration
 
+func pluginIsReadyToBeRemoved(reconciler *ManagedFusionOfferingReconciler, offering *v1alpha1.ManagedFusionOffering) (bool, error) {
```

Comment:
- better have verb at the start of func `pluginIs` vs `isPlugin`

Reply:
the `plugin` prefix is our internal marker for all plugin infrastructure layer callbacks, this is going to change when we introduce an actual plugin abstraction.